css/styles.css

```diff
@@ -0,0 +1,299 @@
+HTML{
+    background-color: rgba(45, 46, 46, 0.767);
+}
+header{
+    background-color: aqua;
+    position: absolute;
+    width: 1920px;
+    height: 840px;
+    left: 0px;
+    top: 0px;
+}
+.JK{
+    position: absolute;
+    width: 556px;
+    height: 109px;
+    left: 682px;
+    top: 296px;
+    
+    font-family: Playfair Display;
```

Comment:
It's a custom font. Use most common standard fonts. And do not forget to define font-placeholders in case if user has no the first font you noted. Look at my project [for example](https://github.com/Dotcommando/adaptive-layout/blob/master/css/styles.css).